service/controller/controller.go

```diff
@@ -67,13 +67,7 @@ func newControllerResourceSets(config ControllerConfig) ([]*controller.ResourceS
 
 	var resourceSet *controller.ResourceSet
 	{
-		c := resourceSetConfig{
-			K8sClient:        config.K8sClient,
-			Logger:           config.Logger,
-			PrometheusClient: config.PrometheusClient,
-
-			BaseDomain: config.BaseDomain,
-		}
+		c := resourceSetConfig(config)
```

Comment:
golangci-lint/ci was complaining with

```go
service/controller/controller.go:70:8: S1016: should convert config (type ControllerConfig) to resourceSetConfig instead of using struct literal (gosimple)
		c := resourceSetConfig{
		     ^
```